### ISSUE TITLE
Reviewer 2, Round 2, 2.3 Uncorrected error in Equation 13

### ISSUE THREAD

**Issue by ramess101** [mostafa-razavi/ITIC-paper#42]
@mostafa-razavi response looks good:

![image](https://user-images.githubusercontent.com/23408516/55347859-026a3d80-5473-11e9-8e06-3ee2522bf320.png)

![image](https://user-images.githubusercontent.com/23408516/55347888-12821d00-5473-11e9-9f61-aeda10330ca0.png)

![image](https://user-images.githubusercontent.com/23408516/55347892-1615a400-5473-11e9-84dd-c8aecb2e273d.png)

**Comment by ramess101** [comment]
@mostafa-razavi  Some minor comments though:

This statement sounds way too combative and should be removed:

> The reviewer seems to believe that this is not allowed, because T is not a constant, however we believe that it is mathematically correct to multiply and divide the integrand by any constant or variable as long as it is not equal to zero. 

I don't think the reviewer ever made such a statement and, even if they did, we should take the high road on this one.

**Comment by ramess101** [comment]
Should this be T1=0?

> Assuming that T!=0, we can integrate TUdep instead of (U-Uig)/R and obtain exact same result.

**Comment by ramess101** [comment]
@mostafa-razavi 

I think the response is good, but because other readers could be confused as well, we probably need to modify the manuscript slightly to clarify how Equation 13 is derived. We should mention that we use Eq 4 with T1=Tsat_old and T2=Tsat_new. We should mention this change at the end of the response. It is always polite to make some modification so that we are not implying that only this reviewer would be confused.

**Comment by ramess101** [comment]
@mostafa-razavi 

For example, I think right here we should point out that Adep_new and Adep_old are obtained from Equation 4 with T1=Tsat_old and T2=Tsat_new:

![image](https://user-images.githubusercontent.com/23408516/55350497-7e678400-5479-11e9-98f9-8ca61d804efc.png)

**Comment by ramess101** [comment]
@mostafa-razavi 

The revised manuscript has a subscript "liq" for Adep but not for Udep. We could probably get away with the "liq" subscript since the sentence before says that we are updating Adep_liq. Or we would need to make this Equation Udep_liq,old and Udep_liq,new. Note that this notation is included in the previous paragraph.

![image](https://user-images.githubusercontent.com/23408516/55521742-45790c00-563f-11e9-9e46-677f6e8da40d.png)

**Comment by mostafa-razavi** [comment]
> The revised manuscript has a subscript "liq" for Adep but not for Udep. We could probably get away with the "liq" subscript since the sentence before says that we are updating Adep_liq. Or we would need to make this Equation Udep_liq,old and Udep_liq,new. Note that this notation is included in the previous paragraph.

Yes. I was kind of hesitant to use "liq" notation here. Adding liq to Udep makes the equation long. So maybe we should revert to previous form.

**Comment by ramess101** [comment]
@mostafa-razavi 

Yeah we should revert this if we aren't going to make Udep "liq" and "vap" too. I think you made some changes for Figure 2 as well, but I think it is fine to use Adep_liq,new and Udep_liq,new there.

**Comment by mostafa-razavi** [comment]
@ramess101 

![image](https://user-images.githubusercontent.com/16358113/55522352-99d9b700-5652-11e9-8eff-6282d862bbdf.png)

**Comment by ramess101** [comment]
Doesnt look terrible to me
